FE/src/components/Account.jsx

```diff
@@ -1,8 +1,412 @@
+              procedure: [Array],
+              notes: [Array],
+              imgurl: 'https://www.foodiesfeed.com/wp-content/uploads/2023/03/close-up-of-butter-chicken-indian-dish.jpg'
+            }
+        }
+    ]
+        
+    useEffect(()=>{
+        async function userCheck () {
+            try {
+                // const response = await fetch(`${API}users/me`, {
+                //     method: "GET",
+                //     headers: {
+                //         "Content-Type":"application/json",
+                //         "Authorization": `Bearer ${token}`,
+                //     },
+                // });
+                // const result = await response.json()             
+                if (token !== null) {
+                    setUserData(dummyUser);
+                    setRecipeList(dummyRecipes);
+                    setReviewList(dummyReviews);
+                    setCommentList(dummyComments);
+                    setFail(false);
+                    setSuccess(true);
+                    // setUserBio(true);
+                    // setUserForm(false);
+                } else {
+                    setFail(true);
+                    setSuccess(false);
+                }
+
+            } catch (error) {
+                setError(error.message);
+                console.log( error );
+            }
+        }userCheck();
+    }, [update])
+
+    async function signIn(){
+        navigate('/login');
+    }
+
+    async function userUpdate() {
+        try {
+            // const response = await fetch(`${API}users/me`, {
+            //     method: "PATCH",
+            //     headers: {
+            //         "Content-Type":"application/json",
+            //         "Authorization": `Bearer ${token}`,
+            //     },
+            //         body: JSON.stringify({
+            //             username,
+            //             email,
+            //             password: registerPassword,
+            //             name,
+            //             imgUrl: userImg,
+            //             admin
+            //         })
+            // });
+            // const result = await response.json()             
+            setUserData(dummyUser);
+            setUpdate((version) => version +1);
+            setUserForm(false);
+            setUserBio(true);
+        } catch (error) {
+            setError(error.message);
+            console.log( error );
+        }
+    }
+
+    useEffect(() => {
+        async function enableButton () {
+            try {
+                if (password == confirmPassword) {
+                    setButtonStatus(false);
+                } else {
+                    setButtonStatus(true);
+                }
+            } catch(error) {
+                console.log(error);
+            }
+        }enableButton();
+    }, [password, confirmPassword]);
 
     return (
-        <><p>This is the Account page</p></>
+        <><p>This is the Account page</p>
```

Comment:
A lot of this account page could probably be split up into individual components, this will make other things easier later, but not really necessary now.